data/icon-overlay.js

```diff
@@ -240,12 +240,15 @@ function getTemplate() {
 
   containerEl.className = 'minvid__overlay__container';
   playIconEl.className = 'minvid__overlay__icon';
+  playIconEl.id = 'minvid__overlay__icon__play';
   playIconEl.title = 'Play Now';
-  addIconEl.className = 'minvid__overlay__icon__add';
+  addIconEl.className = 'minvid__overlay__icon';
+  addIconEl.id = 'minvid__overlay__icon__add';
```

Comment:
Not quite, I removed the fade effect so as long as the mouse is hovered on a video to overlay icons won't fade away. 

However, the description in #439 suggests handling the idle situation where the overlay icons still fade away but will reappear when the mouse is moving (without re-hovering).